app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -484,10 +485,19 @@ private void initViews(@NonNull final PlayerBinding playerBinding) {
         binding.resizeTextView
                 .setText(PlayerHelper.resizeTypeOf(context, binding.surfaceView.getResizeMode()));
 
-        binding.playbackSeekBar.getThumb()
-                .setColorFilter(new PorterDuffColorFilter(Color.RED, PorterDuff.Mode.SRC_IN));
-        binding.playbackSeekBar.getProgressDrawable()
-                .setColorFilter(new PorterDuffColorFilter(Color.RED, PorterDuff.Mode.MULTIPLY));
+        tintThumbRed();
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Please remove these checks, as now the minsdk is lollipop (21), then this PR is ready